IdentityCore/src/webview/response/MSIDWebWPJAuthResponse.m

```diff
@@ -29,4 +29,32 @@
 
 @implementation MSIDWebWPJAuthResponse
 
+- (instancetype)initWithURL:(NSURL *)url
+                    context:(id<MSIDRequestContext>)context
+                      error:(NSError **)error
+{
+    NSString *scheme = url.scheme;
+    NSString *host = url.host;
+    
+    // Check for WPJ or broker response
+    if (!([scheme isEqualToString:@"msauth"] && [host isEqualToString:@"wpj"]))
```

Comment:
just verifying: where do we handle response, when scheme is msauth and host is either "code" or "microsoft.aad.brokerplugin"?

Reply:
Implicitly, it would fail to create WPJ here in this class.
After that, MSIDWebAADAuthResponse would be created to handle it.

For reference I've created #145 , due to importance of broker scenarios.